lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
I've tried both but this not working. I think the reason is that the class is a singleton (class is instantiated statically).
If I monkeypatch `primary_stylesheey_data` and `self.instance` then my code is executed but Opal does not find other methods.
Does that make sense ? Did I miss something ?